sys-apps/firejail/firejail-0.9.62.4.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/netblue30/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/netblue30/firejail.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="Security sandbox for any type of processes"
+HOMEPAGE="https://firejail.wordpress.com/"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="apparmor +chroot contrib debug +file-transfer +globalcfg +network +overlayfs +private-home +seccomp +suid test +userns vim-syntax +whitelist x11"
+RESTRICT="!test? ( test )"
+
+RDEPEND="apparmor? ( sys-libs/libapparmor )"
+
+DEPEND="${RDEPEND}
+	!sys-apps/firejail-lts
+	test? ( dev-tcltk/expect )"
+
+PATCHES=(
+	"${FILESDIR}"/"${P}"-nvidia-no-nogroups.patch
+	"${FILESDIR}"/"${P}"-apparmor-global.patch
```

Comment:
Quote the whole strings? `"${FILESDIR}/${P}-..."`

Reply:
Will do.

Reply:
Wow, so almost nobody does it the way I did, but there's a lot of both:

`"${FILESDIR}"/${P}-foo.patch`
and:
`"${FILESDIR}/${P}-foo.patch"`

...with the former preferred 2:1, if my quick test is accurate (which it might not be):
```
portage-tree $ egrep -h -o -r '[^    ]+FILESDIR.*patch"?' [a-z]* | sed -E 's/[^"]+/_/g' | \
                        sort | uniq -c | sort -rn
  11055 "_"_
   6056 "_"
     94 _"_"
     82 "_"_"_"_
     74 _"_"_
     46 "_"_"_"
     30 _
     16 _"_"_"_"_
     12 "_
     10 "_"_"_"_"_"_
      3 _"_
      3 "_"_"_
      1 _"_"_"_"
      1 _"
      1 "_"_"_"_"_"
```

I'll go with whole-string as you suggested.